src/app/components/audio-recordings/components/analyses-download/analysis-download-row.component.html

```diff
@@ -0,0 +1,50 @@
+<div class="directory-listing-container">
+  <div
+    class="display-contents"
+    [ngStyle]="{ backgroundColor: even ? '#f2f2f2' : 'white' }"
```

Comment:
I was able to get this working without any ng properties by using css's nth-child(2n) psudo attribute.
![image](https://user-images.githubusercontent.com/33742269/211224236-d015b9d1-143a-48fb-a2ff-89ac949244af.png)

This will also resolve the comments bellow relating to `even` attribute, and will greatly simplify the code